integrations/standalone/tests/pageobjects/Section.ts

```diff
@@ -33,6 +33,7 @@ export class Section extends Composite {
   }
 
   private static toggleButtonLocatorInternal(page: Page, label: string) {
-    return page.locator(`.collapsible-trigger:has-text("${label}")`);
+    const regexLabel = new RegExp(`^${label}$`);
+    return page.locator(`.collapsible-trigger:has-text("${label}")`, { hasText: regexLabel });

```

Comment:
Ah yes you are right, I will do it like this:
```
 return page.locator(`.collapsible-trigger`, { hasText: regexLabel });
```
I had to use regex because I added a header called "Accept-Properties" within the REST-Activity and with just (`.collapsible-trigger:has-text("Properties")` there where two results "Properties" and "Accept-Properties.